pkg/requestdecorator/requestdecorator_test.go

```diff
@@ -180,7 +180,7 @@ func TestRequestFactory(t *testing.T) {
 
 	requestFactory := NewRequestFactory(ad, uad)
 
-	if dlen := requestFactory.GetDecorators(); len(dlen) != 2 {
+	if dlen := len(requestFactory.GetDecorators()); dlen != 2 {
 		t.Fatalf("Expected to have two decorators, got %d", dlen)
```

Comment:
The old code is kind of weird. Using %d to print a `[]Decorator` should have produced an interesting result.